### ISSUE TITLE
Luminosity density question

### ISSUE THREAD

**Issue by elehcim** [pynbody/pynbody#493]
Similar to #287. Do we need to add the solar magnitude in the correct band when computing the luminosity density in [derived.py](../blob/448b01c36591b79fa57899ef2ef89c13e852dbad/pynbody/derived.py#L185)?

**Comment by apontzen** [comment]
I don’t think so... it seems to correctly convert a magnitude into a magnitude per unit volume which is what the docs say it do (and how it is used by the image function).

Assuming the starting magnitude is correct, I think this function does what it says on the tin?
